.github/workflows/pre-integration.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        java: [8, 11, 13, 14, 15, 16-ea]
```

Comment:
looks like you accidentally removed the jdk14

Reply:
Yes, but not accidentally :) JDK 14 is pretty dead (JDK 13 and JDK 15 are not). This has the bonus that it does not increase GH actions pipeline time.